actionview/lib/action_view/helpers/url_helper.rb

```diff
@@ -379,48 +380,52 @@ def link_to_unless_current(name, options = {}, html_options = {}, &block)
       end
 
       # Creates a link tag of the given +name+ using a URL created by the set of
-      # +options+ unless +condition+ is true, in which case only the name is
-      # returned. To specialize the default behavior (i.e., show a login link rather
-      # than just the plaintext link text), you can pass a block that
-      # accepts the name or the full argument list for +link_to_unless+.
+      # +options+, unless +condition+ is true. Otherwise only the name is returned or
+      # the given block is yielded. Therefore you can use the block to specialize
+      # the default behaviour, e.g. change the link target (example below).
+      # This behaviour is different from how the block works for +link_to+.
```

Comment:
Unless/otherwise is awkward. How about: "If `condition` is false, creates a link tag [..]. Otherwise, when `condition` is true, [... block with name as fallback]."?